web-local/src/lib/components/column-profile/ColumnSummaryMiniPlots.svelte

```diff
@@ -45,13 +51,43 @@
     containerWidth > COLUMN_PROFILE_CONFIG.compactBreakpoint
       ? formatInteger
       : formatCompactInteger;
+
+  /**
+   * Get the null counts for this profile.
+   */
+  let nullCountQuery;
+  $: if ($runtimeStore?.instanceId)
```

Comment:
No need for the if here. The generated use hooks have the check internally already.

Reply:
Applies to all other such usage

Reply:
got it, thanks. I believe this pattern might be used elsewhere by someone else, was copy-pasting in the interest of time.